src/libifconfig.h

```diff
@@ -77,6 +79,16 @@ ifconfig_errtype ifconfig_err_errtype(ifconfig_handle_t *h);
 /** Retrieves the errno associated with the error, if any. */
 int ifconfig_err_errno(ifconfig_handle_t *h);
 
+/* TODO: should we add an opaque user-provided data pointer ? */
```

Comment:
Yes, that's generally useful in callbacks and if it's not required it's easy to set it to NULL.